databases/week2/DB_W2_HW2.sql

```diff
@@ -0,0 +1,42 @@
+-- Create a new database containing the following tables
+CREATE DATABASE school;
+
+USE school;
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+DROP TABLE class;
```

Comment:
Just as a tip: You can do `DROP TABLE class IF EXISTS`. You could even do `DROP DATABASE school IF EXISTS` before you do `CREATE DATABASE school` that way you can run the entire sql file again and again and get a clean slate each time.